tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -57,7 +57,6 @@ class ModuleScopedSklearnModel(get_model_class()):
     """
     A custom Python model class defined in the test module scope.
     """
-    pass
```

Comment:
Interesting

Reply:
Turns out the `pass` statement is unnecessary